app/controllers/university_controller.rb

```diff
@@ -1,39 +0,0 @@
-# frozen_string_literal: true
-
-class UniversityController < ApplicationController
-  layout "markdown_page"
-
-  def index
-    @page_title = "Open Source Contribution University"
-  end
-
-  def show
-    case params[:id]
-    when "squash"
-      @page_title = "Please squash your commits"
-      set_title(@page_title)
-      set_description("What on earth does squashing commits mean and how do you do it? Find out the what, why, and how of git squashs here.")
-      render 'squash'
-    when "rebase"
-      @page_title = "Please Rebase your commits"
-      set_title(@page_title)
-      set_description("What on earth does rebasing commits mean and how do you do it? Find out the what, why, and how of git rebases here.")
-      render 'rebase'
-    when "reproduction_code", "reproduction", "reproduction_case"
-      @page_title = "Please Provide Reproduction Code"
-      set_title(@page_title)
-      set_description("Get answers for your open source bug reports faster by providing a code that reproduces your problem. Find out how.")
-      render 'reproduction_code'
-    when 'example_app', 'example_apps', 'example-app', 'example-apps'
```

Comment:
This and the above reproduction lines have ids that no longer work. I don't have a good solution offhand except to keep hard-coding them in the controller.